spec/requests/api/v1/course_spec.rb

```diff
@@ -166,36 +166,68 @@ def get_course_codes_from_body(body)
         end
       end
 
+      describe 'response headers'
+      context 'when the recruitment year is in the path' do
+        it 'includes the correct next link' do
+          create(:course,
+                 course_code: "LAST1",
+                 age: 10.minutes.ago,
+                 provider: provider)
 
+          timestamp_of_last_course = 2.minutes.ago
+          _last_course_in_results = create(:course,
+                                           course_code: "LAST2",
+                                           age: timestamp_of_last_course,
+                                           provider: provider)
 
-      it 'includes correct next link in response headers' do
-        create(:course,
-               course_code: "LAST1",
-               age: 10.minutes.ago,
-               provider: provider)
-
-        timestamp_of_last_course = 2.minutes.ago
-        _last_course_in_results = create(:course,
-                                         course_code: "LAST2",
-                                         age: timestamp_of_last_course,
-                                         provider: provider)
+          get '/api/v1/2019/courses',
+              headers: { 'HTTP_AUTHORIZATION' => credentials },
+              params: { changed_since: 30.minutes.ago.utc.iso8601 }
+
+
+          expect(response.headers).to have_key "Link"
+          url = url_for(
+            recruitment_year: 2019,
+            params: {
+              changed_since: timestamp_of_last_course.utc.strftime('%FT%T.%6NZ'),
+              per_page: 100
+            }
+  )
+          expect(response.headers["Link"]).to match "#{url}; rel=\"next\""
+        end
+      end
 
-        get '/api/v1/2019/courses',
-            headers: { 'HTTP_AUTHORIZATION' => credentials },
-            params: { changed_since: 30.minutes.ago.utc.iso8601 }
+      context 'when the recruitment year is in the params' do
```

Comment:
I think I would add a comment explaining why we need to support this in case someone decides to delete it.

Suggested comment text:

> We want to keep legacy support for year as a param in order to maintain backwards compatibility. This will avoid breaking calls from UCAS should they use this older style. The next links we generate used to were of this style, and the UCAS systems were making requests in this style.

Reply:
Done.